changelog/generate.js

```diff
@@ -43,9 +44,18 @@ const getFile = url => new Promise((resolve, reject) => {
     'use strict';
 
     var fs = require('fs'),
-        path = require('path'),
+        path = require('path');
+
+    var tree;
+
+    try {
         // eslint-disable-next-line node/no-missing-require
         tree = require('../tree.json');
+    } catch {
```

Comment:
I guess there _could_ be some other error with the require, so maybe we should add a check in the catch block for the right error type/message?

Alternatively, checking with `fs.existsSync` before the require could also do the trick.

Reply:
Changed to `fs.existsSync` :D